### PR TITLE
Completed french localisation

### PR DESCRIPTION
**Pull request by ZhichGaming**
Co-authored-by: @IsaacMarovitz 

@Wind-Explorer's pull request must be merged first before this PR can be merged, since it uses his revamped localization system and format.

This is a reupload of #103 as in that one, I ran into a few problems.

### CODE REVIEW

**Review by Depal1 — APPROVED** [review]

**Comment by Depal1** [comment]
This PR can be merged as soon as https://github.com/PlayCover/PlayCover/pull/86 is.

**Comment by IsaacMarovitz** [comment]
Will need to be altered slightly to match last-minute changes made to #86 before merging

**Comment by Depal1** [comment]
This PR needs a rebase.

File: PlayCover/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,96 +1,115 @@
...
+
+"button.OK" = "OK";
+"button.Proceed" = "Continuer"; 
+"button.Cancel" = "Annuler";
+"button.Close" = "Fermer"; 
+"button.Dismiss" = "Fermer";
+"button.Install" = "Installer";
+
+"error.corruptedIPA" = "Ce fichier .ipa est corrompu. Il ne contient pas de fichier 'Info.plist'.";
+"error.waitInstallation" = "Veuiller attendre que l'installation actuelle termine.";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:42]: Veuiller -> Veuillez

File: PlayCover/fr.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,96 +1,115 @@
...
+"enable.km" = "Activer le mappage de touches"; 
+"enable.gaming" = "Mode de mappage de clavier amélioré";
+"enable.km.info" = "Permet d'utiliser la souris et le keymap en même temps."; 
+"enable.adaptiveDisplay" = "Affichage adaptatif"; 
+"enable.adaptiveDisplay.info" ="Adapte la fenêtre de l'app pour qu'elle rentre dans votre moniteur.";
+"enable.jbBypass" = "Activer l'évitement Jailbreak (Alpha)"; 
+"enable.jbBypass.info" = "Essaye d'éviter les détections de Jailbreak dans certains jeux (No guarentee to work all the time)";
+
+"displayRefreshRate" = "Taux de rafraîchissement de l'écran";
+"mouseSensitivity" = "Sensitivté de la souris : %.f";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:67]: Sensitivté -> Sensibilité

File: PlayCover/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -1,96 +1,115 @@
...
+
+"settings.reset" = "Réinitialiser réglages et keymap";
+"settings.resetted" = "Les réglages on été réinitialisés";
+
+"setupView.header" = "Suivez les étapes ci-dessous pour résoudre des problèmes courants"; 
+"setupView.subheader" = "Ex: Ne pas pouvoir se connecter, certains applications ne marchent pas";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:73]: certains -> certaines

File: PlayCover/fr.lproj/Localizable.strings [hunk 4]
```diff
@@ -1,96 +1,115 @@
...
+
+"setupView.1" = "1) Éteignez votre Mac"; 
+"setupView.2" = "2) Attendez au moins 20 secondes pour que votre Mac se ferme et ensuite maintenir le bouton d'alimentation enfoncé";
+"setupView.3" = "3) Quand \"Loading Startup Options\" apparait, relachez le bouton d'alimentation"; 
+"setupView.4" = "4) Cliquez l'option Recovery Mode boot"; 
+"setupView.5" = "5) Ouvrez Terminal (Barre de menu -> Utilités -> Terminal)";
+"setupView.6" = "6) Entrez: "; 
+"setupView.7" = "7) Entrez la commande. Ensuite, entrez votre mot de passe (Le texte sera caché)"; 
+"setupView.8" = "8) Clickez sur le symbole Apple dans la barre du menu et redémarrer.";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:82]: Clickez -> cliquez

File: PlayCover/fr.lproj/Localizable.strings [hunk 5]
```diff
@@ -1,96 +1,115 @@
...
+
+"setupView.1" = "1) Éteignez votre Mac"; 
+"setupView.2" = "2) Attendez au moins 20 secondes pour que votre Mac se ferme et ensuite maintenir le bouton d'alimentation enfoncé";
+"setupView.3" = "3) Quand \"Loading Startup Options\" apparait, relachez le bouton d'alimentation"; 
+"setupView.4" = "4) Cliquez l'option Recovery Mode boot"; 
+"setupView.5" = "5) Ouvrez Terminal (Barre de menu -> Utilités -> Terminal)";
+"setupView.6" = "6) Entrez: "; 
+"setupView.7" = "7) Entrez la commande. Ensuite, entrez votre mot de passe (Le texte sera caché)"; 
+"setupView.8" = "8) Clickez sur le symbole Apple dans la barre du menu et redémarrer."; 
+"setupView.9" = "Si vous encontrez un erreur, recommencez le procès.";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:83]: encontrez -> rencontrez

File: PlayCover/fr.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,96 +1,115 @@
...
+"setupView.youtube" = "YouTube";
+"setupView.bilibili" = "BiliBili";
+
+"setupView.pressButtonAndEnterPassword" = "Veuiller désactiver SIP et entrer votre mot de passe.";
+"setupView.pressButtonAndEnterPassword.button" = "Désactiver SIP";
+
+"notification.appInstalled" = "Application installé"; 
+"notification.appInstalled.message" = "Ouvrez-le"; 
+
+"setup.enterLoginPassword" = "Veuiller entrer votre mot de passe";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:95]: Veuiller -> Veuillez

File: PlayCover/fr.lproj/Localizable.strings [hunk 7]
```diff
@@ -1,96 +1,115 @@
...
+
+"setupView.pressButtonAndEnterPassword" = "Veuiller désactiver SIP et entrer votre mot de passe.";
+"setupView.pressButtonAndEnterPassword.button" = "Désactiver SIP";
+
+"notification.appInstalled" = "Application installé"; 
+"notification.appInstalled.message" = "Ouvrez-le"; 
+
+"setup.enterLoginPassword" = "Veuiller entrer votre mot de passe"; 
+"setup.incorrectPassword" = "Mot de passe incorrect. Veuiller réessayer."; 
+"setup.enterLoginPassword.info" = "Entrer le mot de passe utilisé pour déverrouiller votre Mac.";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:97]: Entrer -> Entrez

File: PlayCover/fr.lproj/Localizable.strings [hunk 8]
```diff
@@ -1,96 +1,115 @@
...
+"setupView.pressButtonAndEnterPassword" = "Veuiller désactiver SIP et entrer votre mot de passe.";
+"setupView.pressButtonAndEnterPassword.button" = "Désactiver SIP";
+
+"notification.appInstalled" = "Application installé"; 
+"notification.appInstalled.message" = "Ouvrez-le"; 
+
+"setup.enterLoginPassword" = "Veuiller entrer votre mot de passe"; 
+"setup.incorrectPassword" = "Mot de passe incorrect. Veuiller réessayer."; 
+"setup.enterLoginPassword.info" = "Entrer le mot de passe utilisé pour déverrouiller votre Mac."; 
+"setup.enterLoginPassword" = "Veuiller redémarrer votre Mac pour que les changements prennent effet.";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:98]: Veuiller -> Veuillez

File: PlayCover/fr.lproj/Localizable.strings [hunk 9]
```diff
@@ -1,96 +1,115 @@
...
+"enable.adaptiveDisplay" = "Affichage adaptatif"; 
+"enable.adaptiveDisplay.info" ="Adapte la fenêtre de l'app pour qu'elle rentre dans votre moniteur.";
+"enable.jbBypass" = "Activer l'évitement Jailbreak (Alpha)"; 
+"enable.jbBypass.info" = "Essaye d'éviter les détections de Jailbreak dans certains jeux (No guarentee to work all the time)";
+
+"displayRefreshRate" = "Taux de rafraîchissement de l'écran";
+"mouseSensitivity" = "Sensitivté de la souris : %.f";
+
+"settings.reset" = "Réinitialiser réglages et keymap";
+"settings.resetted" = "Les réglages on été réinitialisés";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:70]: on -> ont

File: PlayCover/fr.lproj/Localizable.strings [hunk 10]
```diff
@@ -1,96 +1,115 @@
...
+
+"logs.copied" = "Registre copié"; 
+
+"enable.km" = "Activer le mappage de touches"; 
+"enable.gaming" = "Mode de mappage de clavier amélioré";
+"enable.km.info" = "Permet d'utiliser la souris et le keymap en même temps."; 
+"enable.adaptiveDisplay" = "Affichage adaptatif"; 
+"enable.adaptiveDisplay.info" ="Adapte la fenêtre de l'app pour qu'elle rentre dans votre moniteur.";
+"enable.jbBypass" = "Activer l'évitement Jailbreak (Alpha)"; 
+"enable.jbBypass.info" = "Essaye d'éviter les détections de Jailbreak dans certains jeux (No guarentee to work all the time)";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:64]: No guarentee to work all the time -> Aucune garantie de fonctionnement

File: PlayCover/fr.lproj/Localizable.strings [hunk 11]
```diff
@@ -1,96 +1,115 @@
...
+"error.reinstallApp" = "Veuillez réinstaller PlayCover";
+
+"menubar.downloadMoreApps" = "Télécharger plus d'apps"; 
+"menubar.downloadMoreApps.help" = "Utilisez ce site pour télécharger des apps"; 
+"menubar.exportToSideloady" = "Exporter pour Sideloadly";
+"menubar.exportToSideloady.help" = "Exporter des IPA pour Sideloady du site sideloady.io qui n'a pas besoin d'avoir SIP désactivé pour que certains apps marchent."; 
+
+"bottomBar.notices" = "Notes";
+"bottomBar.notices.help" = "Nouvelles et annonces des développers";
+"bottomBar.setupViewButton" = "Est-ce que vous rencontrez des problèmes?";
```
Comment (kamui545) [inline PlayCover/fr.lproj/Localizable.strings:52]: Est-ce que vous rencontrez des problèmes? -> Rencontrez-vous des problèmes?

**Comment by ZhichGaming** [comment]
Oh, sorry for all the typos

File: PlayCover/fr.lproj/Localizable.strings (hunk 10 above)
Comment (ZhichGaming) [reply PlayCover/fr.lproj/Localizable.strings:64]: Oh, I totally missed that one

**Comment by ZhichGaming** [comment]
I think all conflicts are resolved? Though for some reason a lot of commits appeared.

**Comment by ZhichGaming** [comment]
Apparently, I have messed up the git tree so I am closing this pr.

**Pull request closed without merging**